lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -1014,6 +1024,8 @@ def build_new_order_xml(action, money, payment_source, parameters = {})
             add_stored_credentials(xml, parameters)
             add_pymt_brand_program_code(xml, payment_source, three_d_secure)
             add_mastercard_fields(xml, payment_source, parameters, three_d_secure) if mastercard?(payment_source)
+            add_third_party_token(xml, parameters) if parameters[:store]
```

Comment:
❓ Question: 
Why are you using a GSF store? `Core` currently allows you to process without that.
💭  Suggestion:
You could create a store method, where this `add_third_party_token` is called, instead of verifying a store GSF.

Reply:
yes, you're right, since the gateway doesn't have a store endpoint, we can have two approach,  in this case in order to align us with the way Orbital handle store, we can send it as a GSF, from the Spreedly perspective we can handle the store method, and thinking about the documentation could be make sense use a store method 🤔 , I will check it